install/Docker/dockerfiles/Dockerfile.odim

```diff
@@ -57,6 +56,7 @@ COPY lib-rest-client /ODIM/lib-rest-client/
 COPY lib-utilities /ODIM/lib-utilities/
 COPY install/Docker/dockerfiles/scripts/add-hosts  /ODIM/add-hosts/
 
-COPY install/Docker/dockerfiles/scripts/build_odim.sh .
-RUN ./build_odim.sh
+COPY install/Docker/dockerfiles/scripts/fetch_odim_pkgs.sh .
+RUN ./fetch_odim_pkgs.sh
```

Comment:
since we building  code in each image   IMHO this not required

Reply:
with this script we are downloading the dependencies for building the code